main.py

```diff
@@ -110,8 +111,11 @@ def _remove_submission_exceeding_threshold(self, submission, service):
         reply = templates.submission_exceeding_threshold.format(
             submission.author.name, service.service_name, service.threshold, service.listeners_count, submission.id
         )
-        submission.mod.remove()
+
         self._reply_with_sticky_post(submission, reply)
+        mod_message = '{} exceeds {:,}.  Actual: {:,}.'.format(service.service_name, service.threshold, service.listeners_count)
```

Comment:
can you add this to the templates?

Reply:
also, is this still necessary as the bot isn't getting shadow banned with the message

Reply:
I'll move it to templates for sure.

The line is still nice (if it ends up working) because it adds a mod note for moderators on the removed post with the removal reason.

Basically, on the mod side instead of simply saying "Removed" it will say "Removed.  Youtube plays exceed 500,000.  Actual: 799,123." So that mods can quickly understand why the removal happened without having to actually visit the post.